src/bdc/steps/regionalatlas.py

```diff
@@ -186,3 +192,36 @@ def get_data_from_address(self, row):
                     break
 
         return return_values
+
+    def calculate_regional_score(self, lead) -> float | None:
+        """
+        Calculate a regional score for a lead based on information from the RegionalAtlas API.
+
+        This function uses population density, employment rate, and average income to compute
+        the buying power of potential customers in the area in millions of euro.
+
+        The score is computed as:
+            (population density * employment rate * average income) / 1,000,000
+
+        Possible extensions could include:
+        - Population age groups
+
+        :param lead: Lead for which to compute the score
+
+        :return: float | None - The computed score if the necessary fields are present for the lead. None otherwise.
+        """
+
+        if (
+            lead[f"{self.name.lower()}_pop_density"] is None
+            or lead[f"{self.name.lower()}_employment_rate"] is None
+            or lead[f"{self.name.lower()}_disp_income_p_inhabitant"] is None
+        ):
+            return None
+
+        regional_score = (
+            lead[f"{self.name.lower()}_pop_density"]
+            * lead[f"{self.name.lower()}_employment_rate"]
+            * lead[f"{self.name.lower()}_disp_income_p_inhabitant"]
+        ) / 1000000
```

Comment:
Magic Number? :D will it result in the regional score being in the range of 0-1?